tests/sentry/utils/performance_issues/test_performance_detection.py

```diff
@@ -205,6 +203,10 @@ def test_calls_detect_slow_span(self):
                     "_pi_transaction",
                     "aaaaaaaaaaaaaaaa",
                 ),
+                call(
+                    "_pi_slow_span_fp",
+                    "1-GroupType.PERFORMANCE_SLOW_SPAN-020e34d374ab4b5cd00a6a1b4f76f325209f7263",
+                ),
```

Comment:
Shoot, forgot to delete this unit tests, its already in test_slow_span_detector.py during the refactor, I can create a quick PR for that

Reply:
Fixed in https://github.com/getsentry/sentry/pull/42617